pandas/core/indexes/interval.py

```diff
@@ -191,10 +192,12 @@ class IntervalIndex(ExtensionIndex):
     _typ = "intervalindex"
 
     # annotate properties pinned via inherit_names
-    closed: str
+    closed: Literal["left", "right", "both", "neither"]
```

Comment:
don't we have an alias for this?

Reply:
I created an alias in `_typing`: `IntervalCloseness`

Let me know if you have a better name in mind.

Reply:
renamed to "IntervalBound"